src/services/purchaseService.ts

```
@@ -84,6 +85,35 @@ export const handlePurchase = async (purchaseRequest: IPurchaseRequest, accountI
             break;
+        case 18:
+            if (purchaseRequest.PurchaseParams.SourceId! != worldState.PrimeVaultTraders[0]._id.$oid) {
+                throw new Error("invalid request source");
+            }
+            const offer =
+                worldState.PrimeVaultTraders[0].Manifest.find(
+                    x => x.ItemType == purchaseRequest.PurchaseParams.StoreItem
+                ) ??
+                worldState.PrimeVaultTraders[0].EvergreenManifest.find(
+                    x => x.ItemType == purchaseRequest.PurchaseParams.StoreItem
+                );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)